openedx/core/djangoapps/oauth_dispatch/adapters/dot.py

```diff
@@ -103,3 +106,29 @@ def get_authorization_filters(self, client_id):
             filters.append(self.FILTER_USER_ME)
 
         return filters
+
+    def create_access_token(self, request, user, scope, client, refresh_token=None):
```

Comment:
Question: Does this override the base implementation?  If so, was it possible to call `super` here so the entire implementation doesn't need to be overridden?

Reply:
Not sure what you mean by base implementation. This was copied over from [auth_exchange](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/auth_exchange/views.py#L110) with the idea that we would refactor into some shared module before merging. We need to be able create a DOT AccessToken (with RefreshToken) outside of the DOT API views that do this.